packages/contracts-bedrock/deploy/019-SystemDictatorInit.ts

```diff
@@ -0,0 +1,133 @@
+import { ethers } from 'ethers'
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+import { awaitCondition } from '@eth-optimism/core-utils'
+import '@eth-optimism/hardhat-deploy-config'
+import 'hardhat-deploy'
+
+import {
+  assertDictatorConfig,
+  makeDictatorConfig,
+  getContractsFromArtifacts,
+} from '../src/deploy-utils'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deployer } = await hre.getNamedAccounts()
+
+  let controller = hre.deployConfig.controller
+  if (controller === ethers.constants.AddressZero) {
+    if (hre.network.config.live === false) {
+      console.log(`WARNING!!!`)
```

Comment:
Perhaps a `printWarning(meta: string)` function would be useful

Reply:
Can do in another PR